litmus/sched_pfair.c

```diff
@@ -1224,6 +1224,7 @@ static int __init init_pfair(void)
 static void __exit clean_pfair(void)
 {
 	if(unregister_sched_plugin(&pfair_plugin)){
+		
```

Comment:
Sorry, I think there has been a misunderstanding. I certainly didn't mean to suggest that you need a blank line after every 'if' statement. Rather, provide some space around operands, etc. Don't write it as:

```
if(foo){
```

It should instead be:

```
if (foo) {
```

Reference: https://github.com/LITMUS-RT/litmus-rt/blob/linux-4.9-litmus/Documentation/CodingStyle#L203